<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetDialog</class>
 <widget class="QDialog" name="ForgetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>忘记密码</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>380</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>145</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入邮箱</string>
   </property>
  </widget>
  <widget class="PasswdLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>245</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入新密码</string>
   </property>
  </widget>
  <widget class="PasswdLineEdit" name="lineEdit_confirm_password">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>295</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请确认新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errorhint">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>95</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_username</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_email</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_verify_code">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>200</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>验证码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_verify_code</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_new_passwd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_password</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_confirm_passwd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_confirm_password</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_verify_code">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>195</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入验证码</string>
   </property>
  </widget>
  <widget class="TimerButton" name="button_get_code">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>196</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_confirm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswdLineEdit</class>
   <extends>QLineEdit</extends>
   <header>custom/PasswdLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TimerButton</class>
   <extends>QPushButton</extends>
   <header>custom/TimerButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_verify_code</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_confirm_password</tabstop>
  <tabstop>button_confirm</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_get_code</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ForgetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
